### PR TITLE
Add RSS widget module dependencies.

### PR DESCRIPTION
**Pull request by ratimid**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -739,6 +740,12 @@
                 <version>${xmpcore.version}</version>
             </dependency>
 
+            <dependency>
```
Comment (ptahchiev) [inline pom.xml:743]: No need for this - it is already defined in spring-boot parent.

File: pom.xml [hunk 2]
```diff
@@ -352,6 +352,7 @@
         <spotbugs.version>4.0.2</spotbugs.version>
         <spotbugs.omitvisitors>SerializableIdiom</spotbugs.omitvisitors>
         <stacktable-webjars.version>1.0.1</stacktable-webjars.version>
+        <spring-boot-starter-data-redis.version>2.3.0.RELEASE</spring-boot-starter-data-redis.version>
```
Comment (ptahchiev) [inline pom.xml:355]: No need for this - it is already defined in spring-boot parent.

File: pom.xml (hunk 1 above)
Comment (ratimid) [reply pom.xml]: Fixed.

File: pom.xml (hunk 2 above)
Comment (ratimid) [reply pom.xml]: Fixed.

**Pull request merged**